src/serialization/pkl2proto.py

```diff
@@ -0,0 +1,149 @@
+import pickle
+import sys
+import numpy as np
+import font_pb2
+
+
+def openPickle(foo):
+    f = open(foo, 'rb')
+    cur = pickle.load(f)
+    return cur
+
+
+def parseDict(d, filename):
+    #this should return the np array that we will be using in the model
+    # chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    chars = [
+        'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N',
+        'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'X', 'Y', 'Z', 'a', 'b', 'c',
+        'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q',
+        'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'zero', 'one', 'two',
+        'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 'space',
+        'exclam', 'quotedbl', 'numbersign', 'dollar', 'percent', 'ampersand',
+        'quotesingle', 'parenleft', 'parenright', 'asterisk', 'plus', 'comma',
+        'hyphen', 'period', 'slash', 'colon', 'semicolon', 'less', 'equal',
+        'greater', 'question', 'at', 'bracketleft', 'bracketright',
+        'asciicircum', 'underscore', 'grave', 'braceleft', 'bar', 'braceright',
+        'asciitilde'
+    ]
+
+    maxCurvePerGlyph = 80
+    maxClosedCurves = 20
+    #TODO add to this list so that we know what to use
+
+    final = np.ones((len(chars), maxClosedCurves, maxCurvePerGlyph, 3, 2))
+    # 3,2 means 3 x,y coords for quadratic bezier curve
+    final = final * 999
```

Comment:
This is more Pythonic:

```suggestion
    final *= 999.
```